README.md

```diff
@@ -1,3 +1,11 @@
+# filecoin.js
+
+## Status
+
+This repository is in a **standby** state. It is not being actively maintained or kept in sync with the libraries it depends on. It may be archived in the near future. If you are interested in updating or maintaining this library, please open an issue or pull request for discussion.
```

Comment:
```suggestion
This repository is in a **standby** state. It is not being actively maintained or kept in sync with the libraries it depends on. It may be archived in the near future. If you are interested in updating or maintaining this library, please open an issue or pull request for discussion.

The [js-lotus-client](https://github.com/filecoin-shipyard/js-lotus-client) suite of libraries may be used to implement some of the features that were intended to be developed here and is under more active maintenance.
```